transports/webrtc-utils/src/fingerprint.rs

```diff
@@ -18,8 +18,12 @@
 // FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 // DEALINGS IN THE SOFTWARE.
 
+use libp2p_core::multihash;
 use sha2::Digest as _;
 use std::fmt;
+
+// feat tokio
```

Comment:
```suggestion
```

Does the comment provide any value beyond what the `cfg` line does?

Reply:
No, I will remove it

Reply:
Fixed